dags/hivemind_etl_helpers/tests/integration/test_github_etl_fetch_raw_commits.py

```diff
@@ -88,7 +91,7 @@ def test_get_single_commit_single_repo_with_from_date(self):
             )
 
         repository_ids = [123]
-        commits = fetch_raw_commits(
+        commits = self.github_commit_extractor._fetch_raw_commits(
```

Comment:
Please name the `github_commit_extractor` to `github_extractor`, as our goal in the future is for all other extraction parts (issue, comment, pull request) of github, we want the `GitHubExtraction` class to inherit from each of those extractors.